src/mir-regionck.md

```diff
@@ -154,6 +155,118 @@ outlives `'static`. Now, this *might* be true -- after all, `'!1`
 could be `'static` -- but we don't *know* that it's true. So this
 should yield up an error (eventually).
 
+### What is a universe
```

Comment:
This is a really useful section. It might be worthwhile to note that the root universe is not unique in some sense, IIUC. There is a root universe for every "instance" of regionck, right? Otherwise, you would have generics in your universe from other items, right?